src/components/appNavigation/TabsView.js

```diff
@@ -126,7 +126,9 @@ const TabsView = React.memo((props: TabViewProps) => {
         </TouchableOpacity>
       )}
       <Appbar.Content />
-      <Appbar.Action icon="menu" onPress={toggleSidemenu.bind(null, store)} color="white" />
+      <TouchableOpacity onPress={toggleSidemenu.bind(null, store)}>
```

Comment:
why arent we using appbar.action?

Reply:
@sirpy for some reason appbar.navigation does not recognize the custom icons added to gooddollar font.